share/transformers/edu_ageconsearch.py

```diff
@@ -1,138 +0,0 @@
-import re
```

Comment:
In the same spirit, I restored the old source config (disabled and with no harvester) so the system knows what to do with the old data.